Azure/sonic-buildimage
# Enable SAI thrift for Mellanox platform

**- What I did**
Enable SAI thrift in RPC image for Mellanox platform

**- How I did it**
Changed MLNX platforms related makefiles to enable SAI thrift

**- How to verify it**
Build and check if syncd process has a "-r" and "-m" flags. 
Check the SAI thrift listening socket. 

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
